File: Cloth-Store-Agency-BE/ct-core-model/src/main/java/com/example/ctcoremodel/CompanyModel.java
package com.example.ctcoremodel;

import lombok.Data;
import lombok.NoArgsConstructor;
import java.util.Date;
@NoArgsConstructor
@Data
public class CompanyModel {
    private String id;
    private String companyid;
    private String name;
    private Date createDate;
    private Date updateDate;
    private String phone;
    private String address;
    private String code;
}

File: Cloth-Store-Admin-FE/src/app/core/apis/Dtos/Company.model.ts
import {DatePipe} from "@angular/common";

export class CompanyModel {
  id!: string | null;
  companyid!: string | null;
  name!: string | null;
  createDate!: DatePipe | null;
  updateDate!: DatePipe | null;
  phone!: string | null;
  address!: string | null;
  code!: string | null;
}
